/home/shawn/development/hg/4.0/include/std/dll.e:279 in function define_c_proc() 
define_c_proc/func: an integer was expected, not a sequence 
    lib = {}
    routine_name = {
                     {43'+'},
                     144846848
                   }
    arg_types = {}
    msg (from inlined routine 'crash' at 36) = <no value>

... called from /home/shawn/development/hg/4.0/tests/t_c_badmachine.e:7 


Public & Export & Global & Local Variables

 /home/shawn/development/hg/4.0/include/std/memconst.e:
    DEP_really_works = 0
    use_DEP = 1
    FREE_RID = 1
    kernel_dll = <no value>
